main.py

```diff
@@ -21,5 +21,10 @@ def read_root():
 
 
 @app.get("/inflation")
-def inflation(region: str = "all"):
-    return {'region': region, 'inflation': inflation}
+def calculate_inflation(region: str = "all"):
+    old_price = 100.0
+    new_price = 120.0
```

Comment:
What these values mean?

Reply:
For example, maybe

Reply:
Okay, but the prices should be taken from the database, agree?